kratos/includes/convection_diffusion_settings.h

```diff
@@ -27,6 +27,7 @@
 
 // Project includes
 #include "includes/define.h"
+#include "includes/variables.h"
```

Comment:
```suggestion
#include "containers/variable.h"
```

Reply:
wasn't this in another PR these days?

Reply:
Yes, most probably I messed something when switching branches. I'll revert in here.

Reply:
I've just checked and it is required in here as well. Indeed, we need to include both `variable.h` and `kratos_components.h` in order to make it work.

Reply:
makes sense to me, still better than `variables.h`

Reply:
I'd cerry-pick the commit in the other PR as well.